pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -146,6 +146,8 @@ type SourceObjectKeySelector struct {
 type KeySelector struct {
 	// Key is the key of the entry in the object's `data` field to be used.
 	Key string `json:"key"`
+	// Arbitrary password for particular store (JKS or PKCS12)
+	Password string `json:"password,omitempty"`
```

Comment:
I don't think you can add this field to the `KeySelector` struct, as it is also used by sources and PEM encoding - that don't have passwords.

Reply:
That was taken into account, if no password was set, then default one is used for JKS, and no one is used for PKCS12. Or do you mean that JKS and pkcs12 are uset as sourses for othe Bundle resources?
If so, what is your proposal?

Reply:
Look at the changes in the generated API spec! You are adding new fields multiple places in the API. I think this new `password` field should only be added to `.spec.target.additionalFormats.jks` and `.spec.target.additionalFormats.pkcs12`. I think you should create a new struct in the API and inline the `KeySelector`:

````go
type Keystore struct {
	KeySelector `json:",inline"`

	Password string `json:"password,omitempty"`
}
````

Then use this struct for the listed fields.

Reply:
OMG, that's true password was many times in the different places, it's my mistake, fixing it.

Reply:
Issue was fixed! Indeed, structure `KeySelector` is used in many places, I didn't take it into account, that was my mistake!

Reply:
Can you please rebase the PR, @arsenalzp. I will try to review tonight.